<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-CL25R4D" ignored="0" name="Failure Test" tests="1" failures="1" timestamp="2022-11-30T00:04:45 IST" time="0.01" errors="0">
  <testcase name="doLogin" time="0.003" classname="testCases.TestFailure">
    <failure type="java.lang.AssertionError" message="Failing the login test">
      <![CDATA[java.lang.AssertionError: Failing the login test
at testCases.TestFailure.doLogin(TestFailure.java:10)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- doLogin -->
</testsuite> <!-- Failure Test -->
